### ISSUE TITLE
Group findings by fault type

### ISSUE THREAD

**Issue by kuleszdl** [kuleszdl/SIFEI#2]

**Comment by friederschueler** [comment]
I think this should be done on client side, so I will mark it as SIFEI issue.

**Comment by kuleszdl** [comment]
Yes and no. I think we have some sort of grouping feature on the SIFCore side, and in an earlier version we had support in SIFEI to group the findings but we dropped this since users were rather confused by the groups.

However, we could re-think grouping based on other criteria such as severity (as e. g. findbugs does with "scariest", "very scary" etc. which is probably more user-intuitive than interpreting a numeric score).

I opened #140 to discuss this so this issue will deal only with a grouping feature based on fault type.
